scripts/gitlab/build_and_test.sh

```diff
@@ -152,3 +166,9 @@ then
         echo "ERROR: failure(s) while running CTest" && exit 1
     fi
 fi
+
+echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+echo "~~~~~ CLEAN UP"
+echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+
+make clean
```

Comment:
I think you need to be inside the build dir when you run make clean, it doesn't seem like it's doing anything right now.

Reply:
gasp!

Reply:
good catch, thanks